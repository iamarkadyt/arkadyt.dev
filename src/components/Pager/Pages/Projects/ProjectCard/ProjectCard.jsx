import React from 'react';
import classes from './ProjectCard.module.css';

const projectCard = (props) => {
    return (
        <div className={classes.container}>
            <p>
                <span className={classes.header}>{props.header}</span>
                <br/><br/>
                {props.children}
                <br/><br/>
                <a>{props.url}</a>
            </p>
            <img src={props.img} alt='' />
        </div>
    );
};

export default projectCard;