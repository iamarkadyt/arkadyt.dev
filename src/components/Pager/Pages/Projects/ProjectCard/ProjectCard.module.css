.container {
    display: flex;
    justify-content: center;
    align-items: center;
    margin: 2vmax;

    font-family: Arial, Helvetica, sans-serif;
}

.container p {
    order: 0;
    flex-basis: 45vw;
    padding: 0 2vw;

    color: #eee;
    text-align: right;
}

.container .header {
    font-weight: bold;
}

.container img {
    order: 1;

    width: 45vmax;
    height: 25vmax;
    padding: 0 2vw;

    object-fit: cover;
}