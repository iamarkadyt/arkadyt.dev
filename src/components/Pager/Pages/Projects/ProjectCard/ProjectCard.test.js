import React from 'react';
import ReactDOM from 'react-dom';
import ProjectCard from './ProjectCard';

it('mounted with no errors.', () => {
    const div = document.createElement('div');
    ReactDOM.render(<ProjectCard />, div);
});