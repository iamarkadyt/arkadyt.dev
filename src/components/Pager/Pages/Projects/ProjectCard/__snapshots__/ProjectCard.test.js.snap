// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot. 1`] = `
<div
  className="container"
>
  <p>
    <span
      className="header"
    >
      Header
    </span>
    <br />
    <br />
    <div
      className="sampleDiv"
    />
    <br />
    <br />
    <a>
      /url
    </a>
  </p>
  <img
    alt=""
    className="image"
    src="/img-link"
  />
</div>
`;
